src/Html.php

```diff
@@ -979,6 +979,7 @@ public static function div(string|Stringable $content = '', array $attributes =
     {
         $tag = Div::tag();
         if (!empty($attributes)) {
+            $attributes = self::filterNullAttributes($attributes);
```

Comment:
Need filter nulled attributes for all tags. See `Html::renderTagAttributes()`

Reply:
Changes have been made. Also corrected styling for ```Html::filterNullAttributes()```